examples/example_pubsub_emulator/README.md

```diff
@@ -0,0 +1,18 @@
+# PubSub Emulator Example
+In the dynamic landscape of contemporary software development, the efficacy of Google Cloud Pub/Sub service is undeniable; however, its seamless integration into development workflows poses challenges in terms of speed, cost, and potential live system disruptions. Addressing these issues, the incorporation of a Pub/Sub emulator in local development emerges as a pivotal practice. By replicating the functions of this service on developers' machines, emulators enable offline work, eliminate financial constraints, expedite testing iterations, and enhance data security, collectively fostering a more efficient and confident development process.
+
+## Running Emulator
+```bash
+docker compose up -d
+export PUBSUB_EMULATOR_HOST="localhost:8085"
+export GOBLET_LOCAL_URL="http://host.docker.internal:8080"
```

Comment:
these exports should be part of the running example right?

Reply:
yes they are here: https://github.com/nthings/goblet/blob/INFRA-1620-local-experience-spanner-goblet-docker-pubsub/examples/example_pubsub_emulator/main.py#L8